README.md

```diff
@@ -5,26 +5,26 @@
 
 #### Compose a cloud native masterpiece.
 
-Infused with cloud native capabilities from the moment you start, Appsody provides everything you need to iteratively develop applications, ready for deployment to Kubernetes environments. Teams are empowered with sharable technology stacks, configurable and controllable through a central hub.
+Infused with cloud native capabilities from the moment you start, Appsody provides everything you need to iteratively develop applications, ready for deployment to Kubernetes environments. Teams are empowered with sharable technology stacks, configurable and controllable through a central hub. To find out more about Appsody check out [appsody.dev](https://appsody.dev).
 
 # Stacks
 
 This repository holds the default set of stacks available from Appsody. Each stack is pre-configured with popular open source technologies to enable rapid development of quality microservice-based applications. Stacks include a base container image and project templates which act as a starting point for your application development.
 
 Stacks allow for rapid development whilst giving the stack developer the ability to control the overall applications that are created from it.
 
-To find out more about Appsody check out [appsody.dev](https://appsody.dev).
+Click here to find out more about [Appsody stacks](https://appsody.dev/docs/stacks/stacks-overview).
 
 ## Repository Structure
 Stacks are categorized as either stable, incubator or experimental depending on the content of the stack.
 
-- `stable/`: Stable stacks meet this set of [technical requirements](https://github.com/appsody/stacks/blob/master/TECHNICAL_REQUIREMENTS.md).
+- `stable/`: Stable stacks are production-ready.
 
-- `incubator/:` The stacks in the incubator folder are actively being worked on to satisfy the stable criteria.
+- `incubator/`: Incubator stacks are actively being worked on to satisfy the stable criteria.
 
-- `experimental/:` Experimental stacks are not being actively been worked on and may not fulfill the requirements of an Appsody stack. These can be used for trying out specific capabilities or proof of concept work.
+- `experimental/`: Experimental stacks are not being actively been worked on and may not fulfill the requirements of an Appsody stack. These can be used for trying out specific capabilities or proof of concept work.
 
-Click here to find out more about [Appsody stacks](https://github.com/appsody/website/blob/master/content/docs/stacks/stacks-overview.md).
+Click here to find out more about [Appsody stacks](https://appsody.dev/docs/stacks/stacks-overview#stack-stability-levels).
```

Comment:
perhaps the link test could include stability levels to make it clearer it is diff from the link above.